test/CMakeLists.txt

```diff
@@ -6,21 +6,23 @@ include(chimeraFunctions)
 find_package(Boost REQUIRED COMPONENTS python)
 find_package(PythonLibs REQUIRED)
 
-include_directories(SYSTEM
-  ${Boost_INCLUDE_DIRS}
-  ${PYTHON_INCLUDE_DIRS}
-)
-
 add_chimera_binding(
   EXCLUDE_FROM_ALL
   TARGET test_binding
   SOURCES dart-example.cpp
   NAMESPACES dart
 )
+
+target_include_directories(test_binding SYSTEM
+  PRIVATE ${Boost_INCLUDE_DIRS}
+  PRIVATE ${PYTHON_INCLUDE_DIRS}
+)
+
 target_link_libraries(test_binding
   ${Boost_PYTHON_LIBRARIES}
```

Comment:
I think you are right according to: https://cmake.org/cmake/help/v3.0/module/FindBoost.html
```
Boost_<C>_LIBRARY      - Libraries to link for component <C> (may include
                         target_link_libraries debug/optimized keywords)
```